pkg/manager/member/utils.go

```diff
@@ -291,9 +290,25 @@ func updateStatefulSet(setCtl controller.StatefulSetControlInterface, tc *v1alph
 		if err != nil {
 			return err
 		}
+		v, ok := oldSet.Annotations[label.AnnStsLastSyncTimestamp]
+		if ok {
+			newSet.Annotations[label.AnnStsLastSyncTimestamp] = v
+		}
```

Comment:
this is not necessary

Reply:
If we remove it, the AnnStsLastSyncTimestamp annotation would be cleaned after each tidbcluster syncing. Though it would not affect adding sts to the sts controller, it would be difficult for us to observe whether the dedicate controller is working when we enable the pod admission webhook.

Reply:
ok, can you move these lines after `set.Annotations = newSet.Annotations`, and set `set.Annotations` instead of `newSet.Annotations`?

`set.Annotations` and `newSet.Annotations` are the same map, but we are going to call `UpdateStatefulSet` on `set`. it's clear.

Reply:
updated.